src/client/opamCommands.ml

```diff
@@ -921,7 +921,17 @@ let show cli =
         available packages as well as a short description for each.";
     `P "Paths to package definition files or to directories containing package \
         definitions can also be specified, in which case the corresponding \
-        metadata will be shown."
+        metadata will be shown.";
+    `P "Note: on terminal supporting it, version numbers can be colored the \
+        following ways:";
+    `Noblank;
+    `I ("-", "Versions installed the current switch are displayed bold and magenta");
+    `Noblank;
+    `I ("-", "Versions installed in other switches are displayed bold");
+    `Noblank;
+    `I ("-", "Versions not available on the current platform are displayed crossed and red");
+    `Noblank;
+    `I ("-", "Versions tagged with the avoid-version or deprecated flag are displayed gray");
```

Comment:
I'm wondering if it is better to invert, like that information is available at first look:
```suggestion
    `I ("-", "Magenta and bold: versions installed the current switch");
    `Noblank;
    `I ("-", "Bold: versions installed in other switches");
    `Noblank;
    `I ("-", "Red and crossed: versions not available on the current platform");
    `Noblank;
    `I ("-", "Gray: versions tagged with the avoid-version or deprecated flag");
```